release-notes/README.md

```diff
@@ -46,6 +46,8 @@ Please note that this release introduces breaking changes. For more details, see
 
 * Fixed an issue where the click notification was not being sent to Adobe Target for a retrieved mbox location upon the `clickedLocation` API call.
 
+* Added support for sending click conversion A4T payload to Adobe Analytics for A4T enabled Target activities upon the `clickedLocation` API call.
```

Comment:
"... for sending the click conversion..." 

"... for A4T-enabled Target activities when the `clickLocation` API is called."